src/shared/components/CallToActionButton/CallToActionButton.types.ts

```diff
@@ -0,0 +1,12 @@
+import { To } from 'history'
+import { MouseEvent, ReactNode } from 'react'
+
+export type ColorVariants = 'red' | 'green' | 'yellow' | 'blue'
+
+export type CallToActionButtonProps = {
+  to?: To
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void
+  icon?: ReactNode
+  colorVariant?: ColorVariants
+  label: string
+}
```

Comment:
I'm not sure if we should create another file just to add these types. We could just keep them inside `CallToActionButton.tsx`

Reply:
Yep, could make sense for bigger component I think